packages/app/src/utils/use-accessibility-options.ts

```diff
@@ -0,0 +1,75 @@
+import { assert } from '@corona-dashboard/common';
+import { isDefined } from 'ts-is-present';
+import { useIntl } from '~/intl';
+
+export interface AccessibilityOptions {
```

Comment:
```suggestion
export interface AccessibilityConfig {
```

or

```suggestion
export interface AccessibilityDefinition {
```

...or something else without options :)